templates/cli/lib/questions.js.twig

```diff
@@ -419,8 +468,96 @@ const questionsLogin = [
                 return "Please enter your password";
             }
             return true;
-        }
+        },
+        when: (answers) => answers.method !== 'select'
     },
+    {
+        type: "search-list",
+        name: "accountId",
+        message: "Select an account to use",
+        choices() {
+            const logins = globalConfig.getLogins();
+            const current = globalConfig.getCurrentLogin();
+
+            const data = [];
+
+            const longestEmail = logins.reduce((prev, current) => (prev && (prev.email ?? '').length > (current.email ?? '').length) ? prev : current).email.length;
+
+            logins.forEach((login) => {
+                if (login.email) {
+                    data.push({
+                        current: current === login.id,
+                        value: login.id,
+                        name: `${login.email.padEnd(longestEmail)} ${current === login.id ? chalk.green.bold('current') : ' '.repeat(6)} ${login.endpoint}`,
+                    });
+                }
+            })
+
+            return data.sort((a, b) => Number(b.current) - Number(a.current))
+        },
+        when: (answers) => answers.method === 'select'
+    },
+];
+const questionGetEndpoint = [
```

Comment:
it seems we already have `questionGetEndpoint` here. We have to remove it from sdks.js